Some minor fixes and additional hooks and filters

alastairR opened the pull request:
Described in issues #389 and #393

navjottomer commented:
Sorry, I can't merge everything from your pull request. It is difficult to test everything. The last time I did it, it made major issues with osclass. It would have been better if these were individual PRs, like a separate PR for fixes.

navjottomer commented:
make a new PR with fixes commits only, it'll be merged right away. For Hooks and filters, I have to see some use cases before they'll be considered. The thing with hooks and filters are once they are added they are difficult to manage.
Thanks for your efforts.

alastairR commented:
No problem. I'll do that.
The hooks and filters are what I found most attractive about osclass. It
made it so much easier to extend the application into areas beyond a
classifieds web site. I've started building an ecommerce site using 5.1, a
modified payment pro plugin and a new plugin I'm building call shop. A beta
is at https://test.structechnique.com. It's still a work in progress but
the additional hooks and filters I've added do make it easier. I created
another variation at mybuy.nz which includes the modified sort hooks and
filters, but that is still on osclass 3.x.

On Mon, 27 Sept 2021 at 20:22, Navjot Tomer ***@***.***>
wrote:

> make a new PR with fixes commits only, it'll be merged right away. For
> Hooks and filters, I have to see some use cases before they'll be
> considered. The thing with hooks and filters are once they are added they
> are difficult to manage.
> Thanks for your efforts.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mindstellar/Osclass/pull/395#issuecomment-927602010>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABS6IRVTL333J42RVSJ5D43UEALUNANCNFSM5EVF5SFQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
